src/middlewared/middlewared/plugins/system.py

```diff
@@ -116,6 +120,11 @@ class Config:
                         f'{schema}.syslogserver',
                         'Port must be in the range of 0 to 65535.'
                     )
+
+        if data['syslog_transport'] == 'TLS':
+            await self.middleware.call('certificate.cert_services_validation', data['syslog_tls_certificate'],
```

Comment:
Can we please add a check for cert revocation status in this method or is it already there ?